Userland/Libraries/LibIPC/Decoder.h

```diff
@@ -149,6 +165,23 @@ class Decoder {
     }
 
 private:
+    // FIXME: Replace all of these shenanigans with `typename VariantTypes...[CurrentIndex]` once C++23 lands
```

Comment:
Too soon? that's still just a proposal (not accepted, but at least clang people are trying it)

Reply:
ok well if you remember to do it... :)